libphal/phal_sbe.C

```diff
@@ -310,8 +310,8 @@ sbeError_t capturePOZFFDC(struct pdbg_target *target)
 		return sbeError_t(exception::SBE_FFDC_NO_DATA);
 	}
 
-	// first store all the data based on slid
-	std::unordered_multimap<uint16_t, std::vector<uint8_t>> mulSlidData;
+	//map - slid, <severty, data>
+	std::multimap<uint8_t, std::pair<uint8_t, std::vector<uint8_t>>> mulSlidData;
```

Comment:
Can we use `unordered multimap` here instead of normal `multimap` as I don't think sorting of data is of prime importance here while searching (as the data can't be really that big which could lead to a hash collision for the unordered multimap)? Advantage of using unordered multimap is that for 99% of time insertion and fetching would be of O(1) constant time